app/assets/stylesheets/template_pdf.scss

```diff
@@ -0,0 +1,21 @@
+@import "./colors";
+.template-title {
+  text-align: center;
+  background-color: #3498db;
+  padding: 1rem;
+  color: #fff;
```

Comment:
im not shure, but i believe that when you use color variables they should be used everywhere